be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/DbActor.scala

```diff
@@ -91,21 +92,53 @@ object DbActor {
 
   def getDuration: FiniteDuration = DURATION
 
-  def apply(): DbActor = DbActor()
+  /**
+   * Creates a new [[DbActor]] which is aware of channels already stored in the db
+   *
+   * @return the newly created [[DbActor]]
+   */
+  def apply(): DbActor = {
+    val laoFolder = new File(s"$DATABASE_FOLDER${Channel.rootChannelPrefix}").toPath
```

Comment:
Please add a mention in the README about that persistent storage (like where data is stored, how to use it, how to reset the state, etc...).

Reply:
Added!